### PR TITLE
Update backend to Plone 6.1b1

### PR DESCRIPTION
**Pull request by sneridagh**
I upgraded the setup to the latest coming from Cookieplone.

@pbauer @ericof @fredvd @davisagli could you please drop an eye on it? I tried the images and they work well.

### CODE REVIEW

File: backend-volto/LICENSE.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-plone6demo Copyright 2022, Plone Foundatiuon
+plone6.demo Copyright 2023, Plone Foundation
```
Comment (davisagli) [inline backend-volto/LICENSE.md:1]:
```suggestion
plone6.demo Copyright 2024, Plone Foundation
```

File: backend-volto/requirements.txt [hunk 2]
```diff
@@ -1,8 +1 @@
 -c constraints.txt
--e src/plone6demo[test]
```
Comment (davisagli) [inline backend-volto/requirements.txt:2]: How does the package get installed then?

File: backend-volto/scripts/create_site.py [hunk 3]
```diff
@@ -45,10 +44,11 @@ def asbool(s):
 payload = {
     "title": "Plone 6 Demo Site",
     "profile_id": _DEFAULT_PROFILE,
-    "extension_ids": ["plone6demo:default", "plone6demo:initial", "plone.volto:demo"],
+    "distribution_name": "default",
+    "extension_ids": ["plone6.demo:default", "plone6.demo:initial", "plone.volto:demo"],
```
Comment (davisagli) [inline backend-volto/scripts/create_site.py:48]: extension_ids is ignored if distribution_name is set. These need to either be dependencies of plone6.demo:default, or installed separately below.

File: backend-volto/setup.py [hunk 4]
```diff
@@ -1,28 +1,29 @@
...
+"""
 
 
 setup(
-    name="plone6demo",
-    version="1.0.0a1",
-    description="Plone 6 Demo Site configuration package.",
+    name="plone6.demo",
+    version="1.0.0a0",
+    description="A new project using Plone 6.",
```
Comment (davisagli) [inline backend-volto/setup.py:18]:
```suggestion
    description="Plone 6 Volto demo site configuration package.",
```

File: backend-volto/setup.py [hunk 5]
```diff
@@ -1,28 +1,29 @@
...
     long_description=long_description,
+    long_description_content_type="text/markdown",
     classifiers=[
+        "Development Status :: 3 - Alpha",
```
Comment (davisagli) [inline backend-volto/setup.py:22]:
```suggestion
```

File: backend-volto/setup.py [hunk 6]
```diff
@@ -35,37 +36,42 @@
     keywords="Python Plone CMS",
     author="Plone Foundation",
     author_email="collective@plone.org",
-    url="https://github.com/collective/plone-6-demo-site",
+    url="https://github.com/collective/plone6.demo",
```
Comment (davisagli) [inline backend-volto/setup.py:39]:
```suggestion
    url="https://github.com/plone/demo.plone.org",
```

File: backend-volto/src/plone6/demo/profiles.zcml [hunk 7]
```diff
@@ -1,13 +1,13 @@
...
     xmlns:genericsetup="http://namespaces.zope.org/genericsetup"
-    i18n_domain="plone6demo"
+    i18n_domain="plone6.demo"
     >
 
   <genericsetup:registerProfile
       name="default"
-      title="Plone 6 Demo Site: Install"
-      description="Policy package to install our site"
+      title="Project Title: Install"
```
Comment (davisagli) [inline backend-volto/src/plone6/demo/profiles.zcml:9]:
```suggestion
      title="Plone 6 Demo Site: Install"
```

File: backend-volto/src/plone6/demo/profiles.zcml [hunk 8]
```diff
@@ -23,12 +23,19 @@
 
   <genericsetup:registerProfile
       name="uninstall"
-      title="Plone 6 Demo Site: Uninstall"
-      description="Uninstall Plone 6 Demo Site setup."
+      title="Project Title: Uninstall"
```
Comment (davisagli) [inline backend-volto/src/plone6/demo/profiles.zcml:26]:
```suggestion
      title="Plone 6 Demo Site: Uninstall"
```

File: backend-volto/src/plone6/demo/profiles.zcml [hunk 9]
```diff
@@ -23,12 +23,19 @@
...
+      description="Uninstall Project Title setup."
```
Comment (davisagli) [inline backend-volto/src/plone6/demo/profiles.zcml:27]:
```suggestion
      description="Uninstall Plone 6 Demo Site setup."
```

File: backend-volto/src/plone6/demo/profiles/default/registry/plone.base.interfaces.controlpanel.IMailSchema.xml [hunk 10]
```diff
@@ -4,6 +4,6 @@
            prefix="plone"
            purge="false"
   >
-    <value key="email_from_name">Plone 6 Demo Site</value>
+    <value key="email_from_name">Project Title</value>
```
Comment (davisagli) [inline backend-volto/src/plone6/demo/profiles/default/registry/plone.base.interfaces.controlpanel.IMailSchema.xml:7]:
```suggestion
    <value key="email_from_name">Plone 6 Demo Site</value>
```

File: backend-volto/src/plone6/demo/profiles/default/registry/plone.base.interfaces.controlpanel.ISiteSchema.xml [hunk 11]
```diff
@@ -4,6 +4,6 @@
            prefix="plone"
            purge="false"
   >
-    <value key="site_title">Plone 6 Demo Site</value>
+    <value key="site_title">Project Title</value>
```
Comment (davisagli) [inline backend-volto/src/plone6/demo/profiles/default/registry/plone.base.interfaces.controlpanel.ISiteSchema.xml:7]:
```suggestion
    <value key="site_title">Plone 6 Demo Site</value>
```

File: backend/LICENSE.md [hunk 12]
```diff
@@ -1,4 +1,4 @@
-plone6demo Copyright 2022, Plone Foundatiuon
+plone6.demo Copyright 2023, Plone Foundation
```
Comment (davisagli) [inline backend/LICENSE.md:1]:
```suggestion
plone6.demo Copyright 2024, Plone Foundation
```

File: backend/requirements.txt [hunk 13]
```diff
@@ -1,8 +1 @@
 -c constraints.txt
--e src/plone6demo[test]
```
Comment (davisagli) [inline backend/requirements.txt:2]: How does it get installed?

File: backend/scripts/create_site.py [hunk 14]
```diff
@@ -38,61 +26,41 @@ def asbool(s):
...
 
 if site_id not in app.objectIds():
-    site = site_api._create_site(
-        context=app, distribution_name=DISTRIBUTION, answers=answers
-    )
+    site = addPloneSite(app, site_id, **payload)
     transaction.commit()
+
+    portal_setup: SetupTool = site.portal_setup
+    portal_setup.runAllImportStepsFromProfile("profile-plone6.demo:default")
```
Comment (davisagli) [inline backend/scripts/create_site.py:63]: It is already installed by the voltodemo distribution

File: backend/src/plone6/demo/profiles.zcml [hunk 15]
```diff
@@ -1,25 +1,32 @@
...
     xmlns:genericsetup="http://namespaces.zope.org/genericsetup"
-    i18n_domain="plone6demo"
+    i18n_domain="plone6.demo"
     >
 
   <genericsetup:registerProfile
       name="default"
-      title="Plone 6 Demo Site: Install"
-      description="Policy package to install our site"
+      title="Project Title: Install"
```
Comment (davisagli) [inline backend/src/plone6/demo/profiles.zcml:9]:
```suggestion
      title="Plone 6 Demo Site: Install"
```

File: backend/src/plone6/demo/profiles.zcml [hunk 16]
```diff
@@ -1,25 +1,32 @@
...
+      description="Package to configure a new Project Title site"
```
Comment (davisagli) [inline backend/src/plone6/demo/profiles.zcml:10]:
```suggestion
      description="Package to configure a new Plone 6 Demo Site"
```

File: backend/src/plone6/demo/profiles.zcml [hunk 17]
```diff
@@ -1,25 +1,32 @@
...
       provides="Products.GenericSetup.interfaces.EXTENSION"
       directory="profiles/default"
       />
 
   <genericsetup:registerProfile
       name="uninstall"
-      title="Plone 6 Demo Site: Uninstall"
-      description="Uninstall Plone 6 Demo Site setup."
+      title="Project Title: Uninstall"
```
Comment (davisagli) [inline backend/src/plone6/demo/profiles.zcml:17]:
```suggestion
      title="Plone 6 Demo Site: Uninstall"
```

File: backend/src/plone6/demo/profiles.zcml [hunk 18]
```diff
@@ -1,25 +1,32 @@
...
+      description="Uninstall Project Title setup."
```
Comment (davisagli) [inline backend/src/plone6/demo/profiles.zcml:18]:
```suggestion
      description="Uninstall Plone 6 Demo Site setup."
```

**Review by davisagli — COMMENTED** [review]
What about the Classic UI demo?

File: backend-volto/requirements.txt (hunk 2 above)
Comment (sneridagh) [reply backend-volto/requirements.txt:2]: It's Cookieplone, you tell me :) but it does get installed.

File: backend-volto/scripts/create_site.py (hunk 3 above)
Comment (sneridagh) [reply backend-volto/scripts/create_site.py:48]: Done.

File: backend/requirements.txt (hunk 13 above)
Comment (sneridagh) [reply backend/requirements.txt:2]: same as above...

**Comment by sneridagh** [comment]
@pbauer @fredvd @ericof could you please any of you take care of updating the classic site?

File: backend-volto/requirements.txt (hunk 2 above)
Comment (ericof) [reply backend-volto/requirements.txt:2]: @davisagli, @sneridagh mxdev and mx.ini.

**Comment by sneridagh** [comment]
@pbauer @davisagli @ericof I will merge this one, if someone has time for upgrading Classic side, please do.

**Pull request merged**